colabs/Wczytanie_danych.ipynb

```diff
@@ -846,6 +894,922 @@
           "execution_count": 13
         }
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Załadowanie nowych baz\n",
+        "\n",
+        "   **W celu załadowania nowych danych należy:**\n",
+        "\n",
+        "\n",
+        "1. Pobrać token API:\n",
+        "  * Wejść w *zespół Aktorzy* na teams\n",
+        "   * Wybrać zakładkę *pliki*\n",
+        "    * Pobrać plik *kaggle.json* - API\n",
+        "2. Utworzyć plik na swoim dysku pod nazwą \"Big Data\"\n",
+        "3. Wrzucić pobrane API na swój dysk\n",
+        "4. Uruchomić kod :)\n",
+        "\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "bfRM2_2Lcdoi"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Wczytajmy bazę oskarów: \"Best Movies Watchlist\"\n"
+      ],
+      "metadata": {
+        "id": "ybdfEcrqRZ0s"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from google.colab import drive\n",
+        "drive.mount('/content/gdrive')\n",
+        "\n",
+        "#%cd /content/gdrive/MyDrive/Big Data/kaggle.json\n",
+        "import os\n",
+        "os.environ['KAGGLE_CONFIG_DIR'] = \"/content/gdrive/MyDrive/Big Data\"\n",
+        "\n",
+        "#from google.colab import drive\n",
```

Comment:
pokasuj komentarze w kodzie.

Reply:
To nie moje zadanie - zadanie Emilki, proszę o informację, czy jej część mam również poprawiać

Reply:
Jakie zadanie?